File: age_verifier.js
function age_verifier(age) {
  if (age <  21) { return console.log(age + " too young"); }
  if (age >= 21) { return console.log(age + " proceed")  ; }
}

function getRandomInt(min, max) {
  return Math.floor(Math.random() * (max - min)) + min;
}

n = 1
while (n <= 10) {
  age_verifier(getRandomInt(1, 35));
  n++;
}

File: age_verifier.rb
def age_verifier(age)
  printf "%-3s %s\n", "#{age}:", "too young" if age <  21
  printf "%-3s %s\n", "#{age}:", "proceed"   if age >= 21
end

10.times do
  age_verifier(rand(1..35))
end
